google-cloud-bigquery-storage/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/storage/scaladsl/BigQueryStorage.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2016-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.googlecloud.bigquery.storage.scaladsl
+
+import akka.NotUsed
+import akka.actor.ClassicActorSystemProvider
+import akka.http.scaladsl.unmarshalling.FromByteStringUnmarshaller
+import akka.stream.alpakka.googlecloud.bigquery.storage.impl.SDKClientSource
+import akka.stream.scaladsl.Source
+import akka.stream.{Attributes, Materializer}
+import akka.util.ByteString
+import com.google.cloud.bigquery.storage.v1.DataFormat
+import com.google.cloud.bigquery.storage.v1.storage.{BigQueryReadClient, CreateReadSessionRequest, ReadRowsResponse}
+import com.google.cloud.bigquery.storage.v1.stream.ReadSession.TableReadOptions
+import com.google.cloud.bigquery.storage.v1.stream.ReadSession
+
+import com.google.cloud.bigquery.storage.v1.stream.{DataFormat => StreamDataFormat}
+
+import scala.concurrent.{ExecutionContextExecutor, Future}
+
+/**
+ * Google BigQuery Storage Api Akka Stream operator factory.
+ */
+object BigQueryStorage {
+
+  private val RequestParamsHeader = "x-goog-request-params"
+
+  def createMergedStreams(
```

Comment:
Can we rename `typed[A]` as `createMergedStreams[A]` and implement an implicit unmarshaller for `ReadRowsResponse.Rows`? So everything can go through this method.

Reply:
The main goal for `createMergedStreams` is to make one stream from the multiple ones and give to the user the choice to use an unmarshaller. 
If it is unnecessary can be removed and leave the typed only as is. 
What do you thing @armanbilge ?

Reply:
> The main goal for createMergedStreams is to make one stream from the multiple ones and give to the user the choice to use an unmarshaller.

I think this makes a lot of sense 👍 If I understand correctly, the only difference between `createMergedStreams` and `typed` is that `typed` uses an unmarshaller, right? So I was thinking, maybe `createMergedStreams` can use an unmarshaller too, and we can provide a default unmarshaller for `ReadRowsResponse.Rows`.

Reply:
Ok will apply changes on this, and let you know